components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1.common/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/common/mappings/APIControllerUtil.java

```diff
@@ -256,6 +262,32 @@ private static void handleEndpointSecurityConfigs(JsonObject envParams, APIDTO i
         }
     }
 
+    /**
+     * This method will add the defined available subscription policies in an environment to the particular imported API
+     *
+     * @param importedApiDto API DTO object to be updated
+     * @param policies       policies with the values
+     * @throws APIManagementException If an error occurs when setting policies
```

Comment:
No exception has been thrown here. Shall we remove this java doc line?

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/9537/commits/9c15461c342b998fcd5fa4be4416416031dde8e6